chainerx_cc/chainerx/cuda/cuda_device.h

```diff
@@ -229,7 +231,7 @@ class CudaDevice : public Device {
     std::shared_ptr<void> AllocatePinnedMemory(size_t bytesize);
 
     // Asynchronous transfer from host to this device, w.r.t. host, using temporary pinned memory.
-    // The current device must be set to this device, prior to calling this function.
+    // The current device must be set to this device, prior to calling this  function.
```

Comment:
This seems like a small mistake. Can you revert it?
```suggestion
    // The current device must be set to this device, prior to calling this function.
```